src/apps/frontend/app.component.tsx

```diff
@@ -20,20 +19,22 @@ export default function App(): React.ReactElement {
   }, []);
 
   return (
-    <DepsProvider deps={{
-      accessService: new AccessService(),
-    }}>
-      <Router>
-        <div className='container'>
-          <Header />
-          <Routes>
-            <Route path='/about' element={<About />} />
-            <Route path='/' element={<Login />} />
-            <Route path='*' element={<NotFound />} />
-          </Routes>
-          <Footer />
-        </div>
-      </Router>
-    </DepsProvider>
+    <BaseWebProvider>
+      <DepsProvider
```

Comment:
We should deprecate `DepsProvider`. We initially thought to use dependency injection and `DepsProvider` knows all about application dependencies, kinda inspired by angular. React has its own context paradigm which is better suited, and hence this should be deprecated